banzai/bias.py

```diff
@@ -113,19 +109,15 @@ def group_by_keywords(self):
     def do_stage(self, images):
 
         for image in images:
-            logging_tags = logs.image_config_to_tags(image, self.group_by_keywords)
-            logs.add_tag(logging_tags, 'filename', os.path.basename(image.filename))
-
             # Subtract the overscan if it exists
             if image.data_is_3d():
                 for i in range(image.get_n_amps()):
                     overscan_level = _subtract_overscan_3d(image, i)
-                    logs.add_tag(logging_tags, 'OVERSCN{0}'.format(i + 1), overscan_level)
-                    self.logger.info('Subtracting overscan', extra=logging_tags)
+                    logger.info('Subtracting overscan', image=image,
+                                extra_tags={'OVERSCN{0}'.format(i + 1): float(overscan_level)})
```

Comment:
I think we can make this a single message at the end with all of the overscn keywords in one set. Right now 4 messages to the logger seems excessive.